uwblueprint/project-read-frontend
# "Add classes" step in create session form

### Notion ticket link



["Add classes" step](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=50a6972952584d4a9789c988f2989fe0)



### Implementation description

- Added a "add a class" page within the create session form that allows the user to create classes with `name`, `location`, `days`, and `facilitator`.

<img width="544" alt="Screen Shot 2021-08-17 at 12 09 19 PM" src="https://user-images.githubusercontent.com/47638847/129761820-ec01d333-8337-42ee-b321-fd9783b9209a.png">



### Steps to test

1. Play around with the form at http://localhost:3000/sessions/create (found on the third step) and make sure that the data is being properly formatted in the following format:

<img width="365" alt="Screen Shot 2021-08-17 at 12 08 50 PM" src="https://user-images.githubusercontent.com/47638847/129761735-e9c774df-ea68-4f65-a20c-e2185ced8de8.png">



### What should reviewers focus on?

- Correct use of props & states, functionality, MCBP (more colours better project 🌈 )

### Checklist
